### DISCUSSION TITLE
Original test of GitHub Discussions

### DISCUSSION THREAD

**Discussion by mcgratta** [firemodels/fds#9993] [Announcements]
This is simply a test of the new GitHub Discussions feature.

**Comment by mcgratta** [comment]
Test follow-up.

**Comment by rmcdermo** [comment]
Interesting...

**Comment by drjfloyd** [comment]
Do we want to try and figure out how to change the banner on the main discussion page to point people to the forum so we don't wind up with having to track two general forums?

**Comment by mcgratta** [comment]
I've seen chatter about how GoogleGroup Discussions can be merged into GitHub Discussions. That might be a tall order. It would be nice to have the Discussion Group and Issue Tracker in one place. I still have trouble convincing people to submit issues.

**Comment by uporersith** [comment]
Test - e-mail particpation

**Comment by rmcdermo** [comment]
Hi. I'm replying to this via email.

**Comment by uporersith** [comment]
It works!
I'm replying via e-mail.

Thanks for the confirmation!

On Sat, Mar 11, 2023 at 10:16 AM Randy McDermott ***@***.***>
wrote:

> Hi. I'm replying to this via email.
>
> On Sat, Mar 11, 2023 at 10:14 AM uporersith ***@***.***>
> wrote:
>
> > Test - e-mail particpation
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/firemodels/fds/discussions/9993#discussioncomment-5276335
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ABBWXQMULLDXD3IPEUCQWYTW3SJE7ANCNFSM5HAAGPJA
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/9993#discussioncomment-5276345>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFAE374T2AQT6NM2GZW5MDTW3SJOHANCNFSM5HAAGPJA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
-Andrew Louie :wq

**Comment by rmcdermo** [comment]
Thanks for testing.  This is good to know.
